modular_skyrat/modules/medical/code/wounds/synth/robotic_blunt.dm

```diff
@@ -0,0 +1,1255 @@
+		if (crowbarred_open)
+			. += ", [span_notice("and is violently torn open, internals visible to the outside")]"
+			use_exclamation = TRUE
+		if (gelled)
+			. += ", [span_notice("with fizzling blue surgical gel leaking out of the cracks")]"
+			use_exclamation = TRUE
+		if (use_exclamation)
+			. += "!"
+
+/datum/wound/blunt/robotic/secures_internals/get_scanner_description(mob/user)
+	. = ..()
+
+	var/to_add = get_wound_status()
+	if (!isnull(to_add))
+		. += "\nWound status: [to_add]"
+
+/datum/wound/blunt/robotic/secures_internals/get_simple_scanner_description(mob/user)
+	. = ..()
+
+	var/to_add = get_wound_status()
+	if (!isnull(to_add))
+		. += "\nWound status: [to_add]"
+
+/datum/wound/blunt/robotic/secures_internals/proc/get_wound_status(mob/user)
+	if (crowbarred_open)
+		. += "The limb has been torn open, allowing ease of access to internal components, but also disabling it. "
+	if (gelled)
+		. += "Bone gel has been applied, causing progressive corrosion of the metal, but eventually securing the internals. "
+
+/datum/wound/blunt/robotic/secures_internals/item_can_treat(obj/item/potential_treater, mob/user)
+	if (potential_treater.tool_behaviour == TOOL_WELDER || potential_treater.tool_behaviour == TOOL_CAUTERY)
+		if (ready_to_resolder)
+			return TRUE
+
+	if (ready_to_secure_internals)
+		if (item_can_secure_internals(potential_treater))
+			return TRUE
+
+	return ..()
+
+/datum/wound/blunt/robotic/secures_internals/treat(obj/item/item, mob/user)
+	if (ready_to_secure_internals)
+		if (istype(item, /obj/item/stack/medical/bone_gel))
+			return apply_gel(item, user)
+		else if (!crowbarred_open && item.tool_behaviour == TOOL_CROWBAR)
+			return crowbar_open(item, user)
+		else if (item_can_secure_internals(item))
+			return secure_internals_normally(item, user)
+	else if (ready_to_resolder && (item.tool_behaviour == TOOL_WELDER) || (item.tool_behaviour == TOOL_CAUTERY))
+		return resolder(item, user)
+
+	return ..()
+
+/// Returns TRUE if the item can be used in our 1st step (2nd if T3) of repairs.
+/datum/wound/blunt/robotic/secures_internals/proc/item_can_secure_internals(obj/item/potential_treater)
+	return (potential_treater.tool_behaviour == TOOL_SCREWDRIVER || potential_treater.tool_behaviour == TOOL_WRENCH || istype(potential_treater, /obj/item/stack/medical/bone_gel))
+
+#define CROWBAR_OPEN_SELF_TEND_DELAY_MULT 2
+#define CROWBAR_OPEN_KNOWS_ROBO_WIRES_DELAY_MULT 0.5
+#define CROWBAR_OPEN_KNOWS_ENGI_WIRES_DELAY_MULT 0.5
+#define CROWBAR_OPEN_HAS_DIAG_HUD_DELAY_MULT 0.5
+#define CROWBAR_OPEN_WOUND_SCANNED_DELAY_MULT 0.5
+/// If our limb is essential, damage dealt to it by tearing it open will be multiplied against this.
+#define CROWBAR_OPEN_ESSENTIAL_LIMB_DAMAGE_MULT 1.5
+// Can crowbar the limb to tear it open, inconsistant without shock protection, but massively increases the chance of normally securing internals
+/datum/wound/blunt/robotic/secures_internals/proc/crowbar_open(obj/item/crowbarring_item, mob/living/user)
```

Comment:
Properly document this, and leave some whitespace between the define and the proc, please.